cfclient/cfapps/cache.go

```diff
@@ -15,13 +15,14 @@ type Cache struct {
 	Collection  map[string]*CFApp
 	WriteBuffer chan *CFApp
 	sync        *sync.RWMutex
+	isUpdating  bool
 }
 
 // NewCache ...
 func NewCache() *Cache {
 	cache := &Cache{
 		Collection:  map[string]*CFApp{},
-		WriteBuffer: make(chan *CFApp, 1024),
+		WriteBuffer: make(chan *CFApp, app.Get().Config.GetDuration("FIREHOSE_CACHE_WRITE_BUFFER_SIZE")),
```

Comment:
As below, i would add a protection here for very low numbers. ex: min(1024,  FIREHOSE_CACHE_WRITE_BUFFER_SIZE)

Reply:
this protections are already at the tile configuration level

Reply:
in the tile.yml configuration . I think this will prevents this